build/template_browserextension.js

```diff
@@ -0,0 +1,19 @@
+( function ( $, mw ) {
+	function loadWhoWroteThat() {
+		var $button = $( <a> )
```

Comment:
Don't we need quotes around the markup?

Reply:
Also, I don't really know what's supposed to happen here, but it would seem we're grabbing every link on the page? Or is this just for the browser extension button?

Reply:
Yeah, I'm an idiot here... I wrote this initially as a placeholder in the task and didn't notice when I copied it over. Ironically, I tested the previous version which was using correct markup. It should have quotes; `$( '<a>' )` creates a <a> element. `$( 'a' )` selects all <a> elements.

Let me redo this part. I'll add a proper temporary behavior until we figure out what kind of button we want here. This is the place where we make a button to turn WhoWroteThat on, so it's a "simple" button placed somewhere in the interface (prateek is working it out still).